frontend/src/utilities/useAcceleratorProfileFormState.ts

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { AcceleratorProfileKind } from '~/k8sTypes';
+import { ContainerResources, Toleration } from '~/types';
+import { UpdateObjectAtPropAndValue } from '~/pages/projects/types';
+import useGenericObjectState from './useGenericObjectState';
+import useReadAcceleratorState, { AcceleratorProfileState } from './useReadAcceleratorState';
+
+export type AcceleratorProfileFormData = {
+  profile?: AcceleratorProfileKind;
+  count: number;
+  useExistingSettings: boolean;
+};
+
+export type UseAcceleratorProfileFormResult = {
+  initialState: AcceleratorProfileState;
+  formData: AcceleratorProfileFormData;
+  setFormData: UpdateObjectAtPropAndValue<AcceleratorProfileFormData>;
+  resetFormData: () => void;
+  loaded: boolean;
+  loadError: Error | undefined;
+  refresh: () => Promise<AcceleratorProfileState | undefined>;
```

Comment:
These 3 props are never used in the code, is there any need to return them?

Reply:
just to expose it if its needed elsewhere. we may use it in the future for another feature.